posthog/tasks/test/test_usage_report.py

```diff
@@ -267,6 +267,18 @@ def _test_usage_report(self) -> List[dict]:
                     "dashboard_tagged_count": 0,
                     "ff_count": 2,
                     "ff_active_count": 1,
+                    "hogql_read_bytes": 0,
+                    "hogql_read_rows": 0,
+                    "hogql_duration_ms": 0,
+                    "hogql_read_bytes_via_api": 0,
+                    "hogql_read_rows_via_api": 0,
+                    "hogql_duration_ms_via_api": 0,
+                    "event_explorer_read_bytes": 0,
+                    "event_explorer_read_rows": 0,
+                    "event_explorer_duration_ms": 0,
+                    "event_explorer_read_bytes_via_api": 0,
+                    "event_explorer_read_rows_via_api": 0,
+                    "event_explorer_duration_ms_via_api": 0,
```

Comment:
Can we actually make some of these queries or mock them somehow so that we know that the are being calculated correctly? Having 0 here is useful to know that the field is being included but doesn't tell us that we're actually counting correctly.

Reply:
That's what I get for being lazy...

Test added! Though not for the API rows, yet.